srcjs/input_binding_select.js

```diff
@@ -55,9 +55,11 @@ $.extend(selectInputBinding, {
     if (data.hasOwnProperty('url')) {
       selectize = this._selectize(el);
       selectize.clearOptions();
+      selectize.clearOptionGroups();
       var thiz = this, loaded = false;
       selectize.settings.load = function(query, callback) {
         var settings = selectize.settings;
+        var thiz2 = this;
```

Comment:
Is `thiz2` really necessary?

Reply:
Good catch. In fact, I can get rid of all of "thiz."

Reply:
I haven't looked at it closely yet, but you might still need `thiz`/`self`. (I'm assuming `thiz` was there for a reason.) You may already know this, but `this` in Javascript doesn't behave the way that it does in other languages.

For example, see https://gist.github.com/wch/b6ae0c7d88285f49c6b328c39f61e8da

Reply:
It looked like the original `thiz` was referring to the binding object, and was calling the `setValue` method. `thiz2` was referring to the selectize object (as per [this example](https://github.com/selectize/selectize.js/issues/151#issuecomment-111056161)) and was calling the `setValue` selectize method. 

However, in all cases I believe `thiz` was unnecessary as we could explicitly use the `selectize` variable declared on line 56.